django_mongodb_backend/functions.py

```diff
@@ -268,10 +270,20 @@ def trunc_time(self, compiler, connection):
     }
 
 
+def kms_credentials(self, provider):  # noqa: ARG001
+    return KMS_CREDENTIALS.get(provider, None)
+
+
+def kms_provider(self):  # noqa: ARG001
+    return getattr(settings, "KMS_PROVIDER", None)
+
+
 def register_functions():
     Cast.as_mql = cast
     Concat.as_mql = concat
     ConcatPair.as_mql = concat_pair
+    ConnectionRouter.kms_credentials = kms_credentials
+    ConnectionRouter.kms_provider = kms_provider
```

Comment:
This file is a list of database functions and `ConnectionRouter` isn't one of them. If we do need this, put it in `routers.py`.